### ISSUE TITLE
我现在使用Mini-InternVL-Chat-2B-V1-5进行finetune了，但用的stage3的配置，请问和1，2的区别在哪里？

### ISSUE THREAD

**Issue by SonGohan757** [OpenGVLab/InternVL#254]

**Comment by mi4da** [comment]
{'loss': 0.0, 'learning_rate': 3.3112582781456954e-07, 'epoch': 0.01}
  0%|          | 3/30200 [01:39<256:54:21, 30.63s/it]warning: The size of tensor a (15968) must match the size of tensor b (52736) at non-singleton dimension 0, input_embeds[selected].shape=torch.Size([15968, 2048]), vit_embeds.shape=torch.Size([52736, 2048])

**Comment by mi4da** [comment]
是因为学习率太小吗

**Comment by mi4da** [comment]
感觉是梯度消失

**Comment by caidou05** [comment]
我用4b模型 也是loss 0  你这个解决了吗

**Comment by Felix0805** [comment]
我用2b模型也是loss=0

**Comment by Mohamed-Dhouib** [comment]
Got the same with the 4B model

**Comment by itay1542** [comment]
I also had this problem and fixed it by setting the correct --conv_style parameter in the training script.
in my case I had to change --conv_style to "internlm2-chat"

**Comment by mi4da** [comment]
我的解决方案是确保图片不要有太过分的长宽比就可，如果有的话就用0 pad到正方形，loss就不为0了，
















在 2024-06-13 16:25:10，"caidou05" ***@***.***> 写道：

我用4b模型 也是loss 0 你这个解决了吗

**Comment by wintercat1994** [comment]
是做的用lora的finetune吗？

**Comment by mi4da** [comment]
不是

在 2024-07-05 15:43:18，wintercat1994 ***@***.***> 写道：


是做的用lora的finetune吗？

**Comment by wintercat1994** [comment]
> 不是 在 2024-07-05 15:43:18，wintercat1994 ***@***.***> 写道： 是做的用lora的finetune吗？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***>

感谢回复，请问是因为现在没有一个可跑通的lora训练2b模型的脚本，所以您采用了全量数据微调？

**Comment by mi4da** [comment]
对的

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年07月08日 09:58 |
| 收件人 | OpenGVLab/InternVL ***@***.***> |
| 抄送人 | mi4da ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [OpenGVLab/InternVL] 我现在使用Mini-InternVL-Chat-2B-V1-5进行finetune了，但用的stage3的配置，请问和1，2的区别在哪里？ (Issue #254) |

不是 在 2024-07-05 15:43:18，wintercat1994 @.> 写道： 是做的用lora的finetune吗？ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.>

感谢回复，请问是因为现在没有一个可跑通的lora训练2b模型的脚本，所以您采用了全量数据微调？
